students/brandon_nguyen/session02/fizzBuzz.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+#Fizz Buzz problem - Week 2 Excercise 1
+#Student: Brandon Nguyen - Au2018
+for i in range(1,101):
+    if (i%3 ==0 and i%5 == 0):
```

Comment:
Minor: No need to surround if conditions with parentheses in Python (it's in fact discouraged). No need to change that here, just remember for your future coding.

Reply:
thank you!

Reply:
Please review my series.py and provide feedback.  I was trying to find a proper way to check args but unable to find a "correct" way to check the parameters for type etc.